### PR TITLE
week4: update journal

### PR DESCRIPTION
**Pull request by garrettbslone**

### CODE REVIEW

File: journals/week4.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+flux.%s.%d.event_tx:%d|C\nflux.%s.%d.event_rx:%d|C\n\
+flux.%s.%d.keepalive_tx:%d|C\nflux.%s.%d.keepalive_rx:%d|C",
+	mod_name, rank, ctx->last_mcs.request_tx,
+	mod_name, rank, ctx->last_mcs.request_rx,
+	mod_name, rank, ctx->last_mcs.response_tx,
+	mod_name, rank, ctx->last_mcs.response_rx,
+	mod_name, rank, ctx->last_mcs.event_tx,
+	mod_name, rank, ctx->last_mcs.event_rx,
+	mod_name, rank, ctx->last_mcs.keepalive_tx,
+	mod_name, rank, ctx->last_mcs.keepalive_rx);
```
Comment (garlick) [inline journals/week4.md:165]: Hah I see what you mean about a higher level interface being useful!

File: journals/week4.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+
+```c
+static  int  cache_store (struct  content_cache  *cache, struct  cache_entry  *e)
+{
+	struct  timespec  *t0;
+	if (!(t0  =  malloc (sizeof (*t0)))) {
+		flux_log (cache->h, LOG_ALERT, "fripp time start");
+	} else {
+		monotime (t0);
+		flux_aux_set (cache->h, "fripp::cc-store", t0, free);
```
Comment (garlick) [inline journals/week4.md:269]: I think you meant to store this in the message?   There is a `flux_msg_aux_set()` and `_get()`.

**Review by garlick — COMMENTED** [review]
Nice work!  I'm getting excited about the content graphs.

**Comment by grondo** [comment]
Nice! I'll merge this so you can get next week's journal going! Thanks @garrettbslone

**Pull request merged**